code/modules/crafting/table.dm

```diff
@@ -86,6 +87,8 @@
 			if(istype(I, /obj/item/weapon/reagent_containers/food/snacks))
 				var/obj/item/weapon/reagent_containers/food/snacks/S = I
 				S.create_reagents(S.volume)
+				feedback_add_details("food_made","[S.name]")
```

Comment:
Recommend you replace with type or some other identifier instead of name, name is often changed by admins or code